docs/schemas.md

```diff
@@ -302,3 +302,18 @@ The following template provides an example with two CTA blocks and a terminal Co
   <!-- INSERT ANY VALID HTML HERE -->
 </div>
 ```
+
+### Desmos Interactive block
+
+#### Description
+
+The Desmos interactive block displays a Desmos graphing calculator. The user can add expressions if the `data-expressions` attribute is set to `true`. The schema user can define where the graph viewport is using `data-bottom`, `data-top`, `data-left`, and `data-right` attributes. The schema user can define the size of the Desmos interactive by passing an integer value into `data-width` and `data-height` attributes. The default is 400 pixles width and height. 
+
+#### Schema definition
+
+The following template provides an example with a Desmos interactive graphing calculator block. 
+
+```html
+<div class="os-raise-ib-desmos" data-wait-for-event="eventnameY"  data-height="500" data-width="600"  data-equations='["(1,2)","x=5"]' data-expressions="true" data-top="50" data-bottom="-50" data-left="-50" data-right="50">
```

Comment:
This just occurred to me, but do you think it might be better to have an attribute for if you don't want expressions versus if you do? The reason being I think it would be nice if the following would render the graphing calculator with the expression editor:

```html
<div class="os-raise-ib-desmos-gc"></div>
```

Versus having to do the following for that case:

```html
<div class="os-raise-ib-desmos-gc" data-expressions></div>
```

Thoughts?

Reply:
> ```
> data-expressions
> ```

This would be better. I did not know you can pass a boolean attribute like this without explicitly giving it the value `true` or `false`. I will make this change.

Reply:
:thinking: - Actually, based on conversations with WW I think it will often be the other way where the student will be prompted to do something using the graphing calculator interactive (e.g. "Use the following to plot a curve that ...").

Reply:
Ok, that makes sense. I will make expressions panel the default and an attribute like `data-no-expressions` show the graph only.